<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/register nodelet_manager">
    <remap from="rgb/camera_info" to="/thorvald_001/kinect2_front_camera/hd/camera_info"/>
    <remap from="depth/camera_info" to="/thorvald_001/kinect2_front_sensor/sd/camera_info"/>
    <remap from="depth/image_rect" to="/thorvald_001/kinect2_front_sensor/sd/image_depth_rect"/>
    
  </node>
</launch>
